### PR TITLE
[Finishes #106065728] sync up our default blocks

### PR DESCRIPTION
**Pull request by Bjvanminnen**
We (unfortunately) have two approaches to specifying default blocks. Some of our newer blocks had only ended up in one of those approaches. This gets them back in sync.

### CODE REVIEW

File: dashboard/app/models/applab.rb [hunk 1]
```diff
@@ -205,7 +210,7 @@ def self.palette
         "functionParams_n": null,
         "callMyFunction": null,
         "callMyFunction_n": null,
-        "return": null
+        "return": null,
```
Comment (joshlory) [inline dashboard/app/models/applab.rb]: Nit: trailing comma

**Comment by joshlory** [comment]
LGTM.  Long-term, can we do something like Artist where an empty set of blocks in Level Builder means inherit all the blocks in `levels.js`?

I think @cpirich originally added the palette in `applab.rb` so level builders could see the blocks they had available, but we may be fragmenting the default palette (since the `.level` files only update to match the model on save).

File: dashboard/app/models/applab.rb (hunk 1 above)
Comment (Bjvanminnen) [inline dashboard/app/models/applab.rb]: Does this matter? If it does, should/can we have linters catch it?

Comment (joshlory) [inline dashboard/app/models/applab.rb]: Whoops sorry, I thought this was JavaScript not Ruby.  Ignore!

**Comment by Bjvanminnen** [comment]
The workflow for level builders here is apparently to start with the full set of blocks (populated by applab.rb) and then whittle it down to the ones they want. If they were to delete everything, we'll fallback to the default in levels.js.

I agree that this isn't an ideal solution. The problem here is that we want to populate text content on LB with JS from apps. This is somewhat different than other apps, where we want to populate a toolbox based on JS from apps.

I discussed with Sarah the fact that when we change our defaults, existing .level files will not change. She understood this, and it sounds like that may be desired (or at the very least acceptable) behavior for them.

File: dashboard/app/models/applab.rb (hunk 1 above)
Comment (Bjvanminnen) [inline dashboard/app/models/applab.rb]: It's JS inside Ruby. It still shouldn't matter on IE9 plus tho, right?

Comment (joshlory) [inline dashboard/app/models/applab.rb]: Looks like trailing commas are valid ECMAScript5, but not valid JSON: http://www.json.org/

Comment (Bjvanminnen) [inline dashboard/app/models/applab.rb]: Yeah, that sounds right. I've never really understood why JSON cares about them.

**Pull request merged**